package.json

```diff
@@ -27,12 +27,26 @@
     "url": "https://github.com/lob/hapi-bookshelf-total-count/issues"
   },
   "homepage": "https://github.com/lob/hapi-bookshelf-total-count#readme",
+  "peerDependencies": {
+    "bookshelf": "0.9.x",
+    "hapi": "11.x"
+  },
```

Comment:
It doesn't work with later versions of Hapi unless you use hapi-qs along with it. With regards to bookshelf, it's restrictive because I'm picking the current version instead of checking through travis ci to see if it works with earlier versions as well.